proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -152,3 +157,13 @@ message AddressStringToBytesRequest {
 message AddressStringToBytesResponse {
   bytes address_bytes = 1;
 }
+
+// QueryAccountAddressByIDRequest is the request type for AccountAddressByID rpc method
+message QueryAccountAddressByIDRequest{
+  int64 id = 1;
+}
+
+// QueryAccountAddressByIDResponse is the response type for AccountAddressByID rpc method
+message QueryAccountAddressByIDResponse {
+  string account_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Would rename this field to `address` simply, which makes it consistent with some other places